otx/mpa/cls/inferrer.py

```diff
@@ -53,6 +54,10 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
         model_builder = kwargs.get("model_builder", None)
         dump_features = kwargs.get("dump_features", False)
         dump_saliency_map = kwargs.get("dump_saliency_map", False)
+        # TODO: It looks like we need to modify that code in an appropriate way.
+        if model_cfg.model.head.get("type", None) == "VisionTransformerClsHead":
```

Comment:
@sungchul2 Due to the unusual output structure of the transformer, I first set dump_feature and dump_saliency_map to False. This may result in some things not working. However, I have no issues with training and exporting. (Of course, I've only checked with the toy dataset, so it seems like a good double check to make sure it's working).

Reply:
I checked there is no issue with `dump_feature` and `dump_saliency_map` = None using cifar10.
But, are these hooks (`FeatureVectorHook`, `ReciproCAMHook`) not needed for ViT models?